connect/logger/LoggerConfig.hx

```diff
@@ -34,8 +34,13 @@ class LoggerConfig extends Base {
      *  One of: `Logger.LEVEL_ERROR`, `Logger.LEVEL_INFO`, `Logger.LEVEL_DEBUG`.
      * @return `this` instance to support a fluent interface.
      */
-    public function level(level: Int): LoggerConfig {
-        this.level_ = level;
+    public function level(level:Dynamic):LoggerConfig {
```

Comment:
While passing an argument of varying type is standard procedure in most dynamically typed languages, it will look very strange when the SDK is used with Java and C#. I suggest leaving the `level` method to receive an integer, and add a `levelName` method which sets the level based on a string.